.github/workflows/daily-boot-iso.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         run: dnf install -y lorax
 
       - name: Run lorax
-        run: lorax -p Fedora -v 34 -r 34 -s http://download.fedoraproject.org/pub/fedora/linux/development/rawhide/Everything/x86_64/os/ -s https://download.copr.fedorainfracloud.org/results/rpmsoftwaremanagement/dnf-nightly/fedora-rawhide-x86_64/ -s https://download.copr.fedorainfracloud.org/results/@rhinstaller/Anaconda/fedora-rawhide-x86_64/ -s https://copr-be.cloud.fedoraproject.org/results/@storage/blivet-daily/fedora-rawhide-x86_64/ /tmp/results
+        run: lorax -p Fedora -v 34 -r 34 -s http://pubmirror2.math.uh.edu/fedora-buffet/fedora/linux/development/rawhide/Everything/x86_64/os/ -s https://download.copr.fedorainfracloud.org/results/rpmsoftwaremanagement/dnf-nightly/fedora-rawhide-x86_64/ -s https://download.copr.fedorainfracloud.org/results/@rhinstaller/Anaconda/fedora-rawhide-x86_64/ -s https://copr-be.cloud.fedoraproject.org/results/@storage/blivet-daily/fedora-rawhide-x86_64/ /tmp/results
```

Comment:
Please add a comment explaining why we don't have mirror list here.

Reply:
*Can* we give it a mirror list? If that works, then that'd be much better of course, then dnf would hopefully try a different mirror. I wasn't aware that lorax supports that (and googling didn't really help). I'll try that then.

Reply:
jkonecny | pitti, ups, my fault! I did not meant mirrorlist but that proxy or load-balancer

Reply:
Added a comment.